TrustModel.md

```diff
@@ -0,0 +1,106 @@
+# Trust Model for Confidential Containers
+A clear definition of trust for the confidential containers project is needed to ensure the 
+components and architecture deliver the security principles expected for cloud native 
+confidential computing. It provides the solid foundations and unifying security principles 
+against which we can assess architecture and implementation ideas and discussions.
+
+The [Trust Modeling for Security Architecture Development article](https://www.informit.com/articles/article.aspx?p=31546) 
+defines Trust Modeling as :
+
+    A trust model identifies the specific mechanisms that are necessary to respond to a specific 
+    threat profile.
```

Comment:
Where do we plan to list all profiles targetted to this project? Can this be linked to or a "TODO: See GitHub issue $url" added?

Reply:
Fresh Issues
#28 Lists documentation required for Trust Model
#29 Lists currents thoughts on Threat Vectors/Profiles to be described (using Trust Model terminology)